phpunit.xml

```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.5/phpunit.xsd"
-     bootstrap="./vendor/autoload.php" colors="true" backupGlobals="false"
+     xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
```

Comment:
🟠 How come we downgraded from 8.5 to 4.1?

Reply:
🤷 changed it to match the PHPUnit version
[7edccbb](https://github.com/may-den/html2pdf/pull/3/commits/7edccbb26ee4b0858d71fd3ac0210db11b33e141)